portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -50,11 +54,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
     FetchOptions fetchOptions = FetchOptions.Builder.withLimit(maxComments);
 
-    ArrayList<String> comments = new ArrayList<>();
+    ArrayList<Comment> comments = new ArrayList<>();
+    
     for (Entity entity : results.asIterable(fetchOptions)) {
-      String commentText = (String) entity.getProperty("text");
+      String email = (String) entity.getProperty("email");
+      String text = (String) entity.getProperty("text");
+      
+      Comment comment = new Comment(email, text);
 
-      comments.add(commentText);
+      comments.add(comment);
```

Comment:
I don't think it helps readability to give this comment a name. I would just inline it:

```
comments.add(new Comment(email, text));
```